libs/designsystem/shared/src/translation/translation.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { Inject, LOCALE_ID } from '@angular/core';
+import { da } from './translations/da';
+import { en } from './translations/en';
+import { Translation } from './translation.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TranslationService {
+  private translations: { [key: string]: Translation } = {
+    da,
+    en,
+  };
+
+  public currentTranslation: Translation = en;
```

Comment:
Could also be "activeTranslation"
```suggestion
  public activeTranslation: Translation = en;
```